src/server/transform.rs

```diff
@@ -261,4 +327,19 @@ mod tests {
             flux::formatter::convert_to_string(&transformed).unwrap()
         );
     }
+
+    #[test]
+    fn test_inject_field_filter() {
+        let fluxscript = r#"from(bucket: "my-bucket")"#;
+        let ast = flux::parser::parse_string("".into(), &fluxscript);
+
+        let transformed =
+            inject_field_filter(&ast, "myField".into()).unwrap();
+
+        let expected = r#"from(bucket: "my-bucket") |> filter(fn: (r) => r._field == "myField")"#;
+        assert_eq!(
```

Comment:
Can we also make a test case where a single pipe transformation already exists...and we are then adding the next transformation? Just as a sanity check on how the stringification (`format_from_js_file`) will handle that particular AST.

Reply:
I _think_ that test would currently fail. This implementation is naive. That fix is later in this chain of patches. I'm unsure what `format_from_js_file` has to do with anything.

Reply:
Cool. As long as we have that fix inbound...since it's required before we can do Schema injection.